src/pages/store/collection/[uuid].tsx

```diff
@@ -0,0 +1,61 @@
+import { Typography } from '@/components/common';
+import { ItemCard, Navbar } from '@/components/store';
+import { config } from '@/lib';
+import { StoreAPI } from '@/lib/api';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PrivateProfile, PublicMerchCollection } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import { getDefaultMerchItemPhoto } from '@/lib/utils';
+import styles from '@/styles/pages/store/collections.module.scss';
+import { GetServerSideProps } from 'next';
+
+interface CollectionProps {
+  user: PrivateProfile;
+  collection: PublicMerchCollection;
+}
+
+const CollectionsPage = ({
+  user: { credits },
+  collection: { title, description, items = [] },
+}: CollectionProps) => {
+  return (
+    <div className={styles.container}>
+      <Navbar balance={credits} showBack />
+      <div className={styles.header}>
+        <Typography variant="h1/bold">{title}</Typography>
+        <Typography variant="h4/regular">{description}</Typography>
```

Comment:
you should specify `component` because `Typography` defaults to using `<div>` elements. It won't affect how it looks, but it's generally better practice for screen readers, reader mode, search engines, etc etc. Something like this:

```suggestion
        <Typography variant="h1/bold" component="h1">{title}</Typography>
        <Typography variant="h4/regular" component="p">{description}</Typography>
```

i think

Reply:
Ok yes user accessibility is so important